applications/launchpad_v2/src/store/app/types.ts

```diff
@@ -0,0 +1,10 @@
+import { ThemeType } from '../../styles/themes'
+
+export type ExpertViewType = 'hidden' | 'open' | 'fullscreen'
+export type ViewType = 'MINING' | 'BASE_NODE' | 'WALLET' | 'ONBOARDING'
+
+export interface AppState {
+  expertView: ExpertViewType
+  view?: ViewType
```

Comment:
we always have a view, therefore this is not optional

Reply:
Not sure whether we will always have a view (or rather we will know what view to render). For example, when we will load the application, at the first render we may do not know what to show to the user (except the splashscreen). For instance, we show the onboarding only to new users. When the application starts, we may need to take some steps to determine what view should be rendered. So I believe that `undefined` is acceptable, and better than any string value.